src/store/modules/players.js

```diff
@@ -106,6 +107,12 @@ const mutations = {
       state.players[index][property] = value;
     }
   },
+  setPronoun(state, { player, pronoun }) {
```

Comment:
This seems redundant, as `setPronoun(player, pronoun)` can be achieved with calling `update(player, "pronouns", pronouns)`
Also note: it's plural `pronouns`, not `pronoun`